icdiff

```diff
@@ -668,6 +668,7 @@ def diff(options, a, b):
         except OSError as e:
             if e.errno == errno.ENOENT:
                 print_meta("error: file '%s' was not found" % e.filename)
+                sys.exit(EXIT_CODE_ERROR)
```

Comment:
I'm not sure if that's inline with want you say it's correct.  At least the test log showed that this could be treated as an error.

Reply:
I don't think we should to exit here.  My memory is that `ENOENT` can be triggered in relatively ordinary circumstances?

Reply:
Hm...  That's a good question.  I'm mostly a Linux programmer and I recall that there that error code does not show up unless it's a hard error and the file does not exist.  On Windows, it seems that errno is rough translation of the Windows error code and so there things might be different.  macOS: absolutely no clue...

Reply:
OK, looking more I'm not seeing anything to back up my vague impression, so let's keep the change.